src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py

```diff
@@ -133,7 +133,15 @@ def get_nscf_builder(  # pylint: disable=too-many-arguments
 
     parameters["SYSTEM"]["nbnd"] = nbnd
 
+    # One can either use `nscf` mode or `bands` mode. For the `nscf` mode,
+    # the `nosym` and `noinv` must be set to True to avoid pw.x automatically
+    # adds more (symmetry-equivalent) kpoints to the user-specified kpoints.
+    # For the `bands` mode, pw.x will use exactly what the user provided.
```

Comment:
Why would one want to use`nscf` mode then?

Reply:
In that case, maybe we _don't_ want to make these changes here, but rather set these inputs in the protocol of the `EpwWorkChain` (once the overrides work as intended ^^).

Reply:
emm, indeed maybe that's a better choice. Then we have to first make the overrides working 😅

Reply:
I'll see if I can get that done. ^^